content/en/blog/2022/how-to-get-started-with-opentelemetry-go.md

```diff
@@ -0,0 +1,435 @@
+---
+title: Instrument your Go web application with openTelemetry
+linkTitle: Get Started with OpenTelemetry & Go
+date: 2022-06-27
+author: Team Aspecto
```

Comment:
sorry for being so pedantic, but this is another one I'd like to call out: Until now we have blog posts from individuals or community teams, but not a vendor. I would prefer having the name of the author here, with a link to their github profile, or we can do `author: Naveh Mevorach (Aspecto)` ? (cc: @cartermp, @chalin, @austinlparker)

Reply:
@NavehMevorach, please give the others some time to respond as well.

Reply:
+1